src/layouts/partial-cards.html

```diff
@@ -321,7 +321,8 @@
     }
 
     isView(currentView, defaultView) {
-      return currentView || defaultView;
+      return (currentView || defaultView) &&
+        this.hass.states[currentView || DEFAULT_VIEW_ENTITY_ID];
```

Comment:
Now you're returning a state object instead of just the string as was done before?

Reply:
defaultView is also an object, I only need the know if it exists or not